content/releases.json

```diff
@@ -0,0 +1,239 @@
+{
+    "releaseInfoTemplate":
+    {
+        "name":"", 
+        "version":"",
+        "releaseDate":"", 
+        "downloadLink":"", 
+        "GPGsig":"", 
+        "changeLog":"", 
+        "PR":"", 
+        "comments":[""]
+    },
+
+    "stableReleases":{
+        "latestRelease": [{
+            "name":"libevent-2.1.12-stable.tar.gz", 
+            "version":"2.1.12",
+            "releaseDate":"2020-07-05", 
+            "downloadLink":"https://github.com/libevent/libevent/releases/download/release-2.1.12-stable/libevent-2.1.12-stable.tar.gz", 
+            "GPGsig":"https://github.com/libevent/libevent/releases/download/release-2.1.12-stable/libevent-2.1.12-stable.tar.gz.asc", 
+            "changeLog":"https://raw.githubusercontent.com/libevent/libevent/release-2.1.12-stable/ChangeLog", 
+            "PR":"https://github.com/libevent/libevent/pull/1045"
+        }],
+
+        "otherStableReleases":[
```

Comment:
Let's keep all releases in one key? And just parse the type of the release from the version name